cypress/support/pages/home.pageObject.js

```diff
@@ -4,7 +4,7 @@ class HomePageObject extends PageObject {
   url = '/#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return this.getByDataCy('username-link');
```

Comment:
Ensure that the `getByDataCy` method is correctly implemented in the `PageObject` class. This method should return the element with the specified `data-cy` attribute.